common/tests/security/pr_floodattack.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+_init() {
+
+    # The functions verifies if the device is an ALFA capable of monitor mode. 
+	# Upon verfication it then puts the device into monitor mode to carry out atttacks	
+  
+    echo "Init called"
+	
+	# Check if it is an ALFA device
+	# Checks if it is an ALFA device by comparing the OUI reference
+
+	addr=$(iw $interface info | grep addr | awk '{print $2}')
+	if [ ${add:0:8}=="00:c0:ca" ]; then
+		echo "Given Interface has a ALFA device, we can proced"
+	else
+		echo "Given Interface is not a ALFA device, check your connection"
+		exit 1
+	fi
+	
+	# Check if Monitor mode is already enabled, If not enable Monitor mode
+
+	_init_result
+	
+	if [ "$result" -eq "1" ]; then
+	
+		echo "Device already in monitor mode"
+	
+	else
+		airmon-ng check kill
+		airmon-ng start $interface
```

Comment:
should we use airmon-ng here? 
or leave it more generic with `iwconfig`
@govindsi ??

Reply:
Have changed to use the `iwconfig`